### ISSUE TITLE
Add GradientBoostingClassifier with loss='exponential' as an alternative

### ISSUE THREAD

**Issue by fzr72725** [chrisalbon/notes#1]
Hi Chris,

I just discovered your site today, and I just love it! I really really appreciate that you made it so that people like me can benefit from the convenient notes and flashcards!

As I was reading the page for [AdaboostClassifier](https://chrisalbon.com/machine_learning/trees_and_forests/adaboost_classifier/), I thought it might be a good idea to mention GradientBoostingClassifier as an alternative since according to [sklearn's documentation](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html), when loss='exponential', gradient boosting recovers the AdaBoost algorithm.

Update: Recently I did a project using gradient boosting for multi-classification. And I realized that the approach I mentioned above would not work in this case. Turns out, when loss is set to 'exponential', sklearn's GradientBoostingClassifier can only work for binary classification.

Best,
Z
